pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -29,11 +29,19 @@ import (
 // ClusterServiceClasses for use in the service catalog.
 type ClusterServiceBroker struct {
 	metav1.TypeMeta `json:",inline"`
+
 	// Non-namespaced.  The name of this resource in etcd is in ObjectMeta.Name.
+	// More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata
+	// +optional
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   ClusterServiceBrokerSpec   `json:"spec"`
-	Status ClusterServiceBrokerStatus `json:"status"`
+	// Spec defines the behavior of the broker.
+	// +optional
+	Spec ClusterServiceBrokerSpec `json:"spec,omitempty"`
+
+	// Status represents the current status of a broker.
+	// +optional
+	Status ClusterServiceBrokerStatus `json:"status,omitempty"`
```

Comment:
My big takeaway from this PR is that a TON of stuff that I would not have expected to be optional is in fact optional from an API standpoint.